schema/JF_TRAN_PAC_MEMO.json

```diff
@@ -283,7 +286,7 @@
         "TYPE": "NUM-8",
         "REQUIRED": "X (error)",
         "SAMPLE_DATA": 20120615,
-        "VALUE_REFERENCE": "YYYYMMDD",
+        "VALUE_REFERENCE": null,
```

Comment:
Why was this changed from YYYYMMDD to null?

Reply:
Not entirely sure, to be honest...